android/app/src/main/AndroidManifest.xml

```diff
@@ -165,14 +173,6 @@
         <meta-data
             android:name="com.google.firebase.messaging.default_notification_color"
             android:resource="@color/breez_notification_color" />
-        <!-- Starting in Android 8.0 (API level 26), all notifications must be assigned to a channel
-             or they don't appear. This lets users disable specific notification channels for your app
-             instead of disabling all your notifications. Users can control the visual and auditory
-             options for each channel from the Android system settings, as shown in figure 11.
-             Users can also touch & hold a notification to change behaviors for the associated channel. -->
-        <meta-data
-            android:name="com.google.firebase.messaging.default_notification_channel_id"
-            android:value="@string/offline_payments_notification_channel_id" />
```

Comment:
We're only interested in showing foreground service status & payment received notifications and we're registering only to their notification channels. This part is no longer needed.